tools/manifest/manifest.py

```diff
@@ -127,7 +140,7 @@ def iterdir(self, dir_name: Text) -> Iterable[ManifestItem]:
                 if path[:tpath_len] == tpath:
                     yield from tests
 
-    def update(self, tree: Iterable[Tuple[Text, Optional[Text], bool]], parallel: bool = True) -> bool:
+    def update(self, tree: Iterable[Tuple[Text, Optional[Text], bool]], parallel: bool = True, is_spec: bool = False) -> bool:
```

Comment:
I would pass in the update_func as an argument instead of is_spec. In case we ever add a third manifest type, the caller can easily pass in the right one.